crates/flox-rust-sdk/src/models/publish.rs

```diff
@@ -226,22 +231,29 @@ impl<'flox> Publish<'flox, NixAnalysis> {
         self.analysis.deref()
     }
 
+    pub async fn build(&self) -> Result<(), PublishError> {
+        let nix = self.flox.nix(Default::default());
+        let command = Build {
+            installables: [self.installable()].into(),
+            ..Default::default()
+        };
```

Comment:
Do we need to add the same `--override-input`s as in `get_drv_metadata`?

Reply:
Same question for `sign_binary()`. I wonder if it would be less error prone to use store_paths

Reply:
Yeah here we need that, because we're not in the `crates/flox-rust-sdk/src/actions/package.rs` module which had a helper for that iirc.

sign binary and upload binary would only need that if we do use the `^*` installable (since that will cause an evaluation, unlike store paths)
Maybe that's a reason _against_ using `^*` there?

Reply:
generally we should set that default override earlier 🤔

Reply:
Any downside to using the package module?

Reply:
Oh yeah that's true. Since analyze is the first step and calculates store paths, what about using store  paths for every step afterwards? That would be (marginally) faster anyways, right?

Reply:
Yeah i just made them all use the installable, rather prefer stop changing things for changing them, if you agree i'd go with how it is now, and we can look at store paths when we have flox-types integrated properly, which would make accessing them safer as well